doc/man/8/ceph-disk.rst

```diff
@@ -97,6 +107,13 @@ Usage::
 If the option :option:`--no-start-daemon` is given, the activation
 steps are performed but the OSD daemon is not started.
 
+The latest option :option:`--reactivate` can re-activate the osd which is on the
```

Comment:
s/which is on the deactivate status/which has been deactivated with the deactivate subcommand/